README.md

```diff
@@ -12,9 +12,12 @@ This library generates changelog for repositories using [gitmoji](https://gitmoj
 
 **Workflow:**
 1. Make changes and commit
+  ```
+  git commit . -m ":sparkles: my awesome feature"
```

Comment:
`-am` is the same as `. -m` ?

Reply:
Don't know, I'm always using it. It seems that it is quite the same.